include/hpp/manipulation/path-planner/states-path-finder.hh

```diff
@@ -193,14 +193,55 @@ class HPP_MANIPULATION_DLLAPI StatesPathFinder : public core::PathPlanner {
 
   /// Step 4 of the algorithm
   void preInitializeRHS(std::size_t j, Configuration_t& q);
-  bool analyseOptimizationProblem(const graph::Edges_t& transitions);
-  bool analyseOptimizationProblem2(const graph::Edges_t& transitions,
-                                   core::ProblemConstPtr_t _problem);
+  bool analyseOptimizationProblem(const graph::Edges_t& transitions,
+                                  core::ProblemConstPtr_t _problem);
 
   /// Step 5 of the algorithm
   void initializeRHS(std::size_t j);
   bool solveOptimizationProblem();
 
+  // Structure used to store right hand side constraints and name in form of
+  // hash numbers
+  typedef std::unordered_map<size_t, size_t> Hashmap;  // rhs, name
```

Comment:
To be consistent with the rest of the code, typedefs are suffixed with  `_t`. Moreover, could you use a more expressive name and give an example in the comment ?